get_routers/get_routers.py

```diff
@@ -0,0 +1,83 @@
+"""
+The following Python code sample shows how to access the
+"routers" endpoint of the NCM REST API using the
+Python "requests" module with paging. It makes use of NCM
+and CP credential headers to authenticate and access the
+router data. The router_id is the key and the router_group,
+router_name, and router_state are the values.
+"""
+
+import json
+import requests
+from datetime import datetime, timezone, timedelta
+
+headers = {
```

Comment:
There should be a comment noting that these values need to be substituted by the caller.  It would be nice if we had a searchable keyword in each of these comments, so a customer could easily see all the spots that need to be updated in the sample script.  Maybe something like "# SUBSTITUTE: your API key".  There may be a better keyword than that; the important thing is to be consistent and searchable.

Reply:
# these values need to be substituted by the caller.
headers = {
    'X-CP-API-ID': 'Your CP-API-ID goes here',
    'X-CP-API-KEY': 'Your CP-API-KEY goes here',
    'X-ECM-API-ID': 'Your ECM-API-ID goes here',
    'X-ECM-API-KEY': 'Your ECM-API-KEY goes here',
    'Accept': '*/*',
    'Content-Type': 'application/json',
}